Sources/MapboxDirectionsCLI/ProcessCommand.swift

```diff
@@ -53,6 +56,29 @@ class ProcessCommand<ResponceType : Codable, OptionsType : DirectionsOptions > :
                let jsonData = try? JSONSerialization.data(withJSONObject: object, options: [.prettyPrinted]) {
                 outputText = String(data: jsonData, encoding: .utf8)!
             }
+        case .gpx:
+            var gpxText: String = String("<?xml version=\"1.0\" encoding=\"UTF-8\"?>")
+            gpxText.append("\n<gpx xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xmlns=\"http://www.topografix.com/GPX/1/1\" xsi:schemaLocation=\"http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd\" version=\"1.1\">")
+            
+            guard let routeResponse = routeResponse else { return }
+            guard let routes = routeResponse.routes else { return }
+            
+            let dateFormatter = DateFormatter()
+            dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ss"
+            var time = Date()
+            
+            routes.forEach { route in
+                let shape = route.shape
+                let timeInterval = TimeInterval(route.distance/route.expectedTravelTime)
+                for coord in shape!.coordinates {
+                    gpxText.append("\n<wpt lat=\"\(coord.latitude)\" lon=\"\(coord.longitude)\">")
+                    gpxText.append("\n\t<time> \(dateFormatter.string(from: time))Z </time>")
```

Comment:
Do all coordinates have the same timestamp?

Reply:
Nope. The `timeInterval` is added at the end of the inner loop.

Reply:
Oh, I see. But something looks wrong... That time interval stores an average speed actually. Should we divide estimated duration on coordinates count instead? Or can we use steps here?